### PR TITLE
created new endpoint to get all admin users

### PR DESCRIPTION
**Pull request by djung335**
created a new get endpoint to return all users with the admin role!

### CODE REVIEW

File: src/users/users.service.ts [hunk 1]
```diff
@@ -30,4 +30,12 @@ export class UsersService {
       throw new InternalServerErrorException(error.message);
     }
   }
+
+  async getAllAdmins(): Promise<User[]> {
+    console.log(
```
Comment (onb6) [inline src/users/users.service.ts:35]: You can remove this console.log, in general we don't want console statements in our code so in the future if you use them for debugging purposes, make sure to delete them when you are done

File: src/users/users.service.ts [hunk 2]
```diff
@@ -30,4 +30,12 @@ export class UsersService {
       throw new InternalServerErrorException(error.message);
     }
   }
+
+  async getAllAdmins(): Promise<User[]> {
```
Comment (onb6) [inline src/users/users.service.ts:34]: Can you add a JS doc comment to describe what this function does. I know the others don't have this but I want to standardize this going forward

**Review by onb6 — COMMENTED** [review]
Left a few comments, once those small things are fixed this should be good to go! Great work on this!!

File: src/users/users.service.ts (hunk 2 above)
Comment (djung335) [reply src/users/users.service.ts:38]: feedback implemented!

File: src/users/users.service.ts (hunk 1 above)
Comment (djung335) [reply src/users/users.service.ts]: feedback implemented!

**Review by onb6 — APPROVED** [review]
Woot!

File: src/users/users.service.ts [hunk 3]
```diff
@@ -30,4 +30,12 @@ export class UsersService {
       throw new InternalServerErrorException(error.message);
     }
   }
+
+  /**
+   *
```
Comment (rymaju) [inline src/users/users.service.ts:35]: ⛏️ Can we remove this blank line 😅 
```suggestion
```

File: src/users/users.service.ts [hunk 4]
```diff
@@ -30,4 +30,12 @@ export class UsersService {
...
+   * grabs all users with the role admin and returns them as an array of Users
```
Comment (rymaju) [inline src/users/users.service.ts:36]: 🔧 This isn't a great purpose statement, it more or less restates the signature and the name of the function. How about:

```suggestion
   * Finds all users with the admin role.
```

File: test/users.e2e-spec.ts [hunk 5]
```diff
@@ -28,13 +38,11 @@ describe('Users e2e', () => {
     app = moduleFixture.createNestApplication();
 
     await app.init();
-    await usersRepository.clear();
-    await usersRepository.save(initialAdminUser);
   });
 
   beforeEach(async () => {
     await usersRepository.clear();
```
Comment (rymaju) [inline test/users.e2e-spec.ts:44]: 🤔 Might be merge conflicts with @onb6 's PR

**Review by rymaju — CHANGES_REQUESTED** [review]
Gonna be pedantic and ask for a nicer purpose statement, otherwise looks good!

File: src/users/users.service.ts (hunk 3 above)
Comment (djung335) [reply src/users/users.service.ts]: yes sir

File: src/users/users.service.ts (hunk 4 above)
Comment (djung335) [reply src/users/users.service.ts]: I'll try and write better purpose statements next time

File: test/users.e2e-spec.ts (hunk 5 above)
Comment (djung335) [reply test/users.e2e-spec.ts:44]: uh oh

Comment (onb6) [reply test/users.e2e-spec.ts:44]: It should be fine, I can help with resolving those if necessary.

**Review by rymaju — APPROVED** [review]
LGTM 💯 
Fix merge conflcits and we should be good

**Pull request merged**